@app/client/src/pages/forgot.tsx

```diff
@@ -7,20 +7,27 @@ import React, {
   useMemo,
 } from "react";
 import { promisify } from "util";
-import SharedLayout from "../layout/SharedLayout";
+import SharedLayout, { SharedLayoutChildProps } from "../layout/SharedLayout";
 import { NextPage } from "next";
 import Link from "next/link";
 import { Form, Icon, Input, Button, Alert } from "antd";
 import { FormComponentProps, ValidateFieldsOptions } from "antd/lib/form/Form";
 import { useForgotPasswordMutation } from "@app/graphql";
 import { ApolloError } from "apollo-client";
 import { getCodeFromError, extractError } from "../errors";
+import { Redirect } from "@app/components";
 
 const ForgotPassword: NextPage = () => {
   const [error, setError] = useState<Error | ApolloError | null>(null);
   return (
     <SharedLayout title="Forgot Password">
-      <WrappedForgotPasswordForm error={error} setError={setError} />
+      {({ currentUser }: SharedLayoutChildProps) =>
```

Comment:
The `: SharedLayoutChildProps` cast shouldn't be required?
```suggestion
      {({ currentUser }) =>
```

Reply:
Yes, that's right. I've updated it